pkg/cmd/get.go

```diff
@@ -146,6 +171,38 @@ func (o *GetOptions) Run() error {
 	objs = append(objs, clusterResources...)
 	objs = append(objs, kubearchiveResources...)
 
+	if o.Output != "" {
+		type List struct {
+			Items      []*unstructured.Unstructured `json:"items"`
+			ApiVersion string                       `json:"apiVersion" yaml:"apiVersion"`
+			Kind       string                       `json:"kind"`
+			Metadata   map[string]any               `json:"metadata"`
+		}
+		list := &List{
+			ApiVersion: "v1",
+			Kind:       "List",
+			Metadata:   map[string]any{},
+		}
+		for _, obj := range objs {
+			list.Items = append(list.Items, &obj)
+		}
+		if o.Output == "json" {
+			listData, err := json.MarshalIndent(list, "", "    ")
+			if err != nil {
+				return err
+			}
+			fmt.Println(string(listData))
+			return nil
+		} else if o.Output == "yaml" {
+			listYaml, err := yaml.Marshal(list)
+			if err != nil {
+				return err
+			}
+			fmt.Println(string(listYaml))
+			return nil
+		}
+	}
```

Comment:
If you have enums for the output types, this whole if clause can be replaced with a switch statement. It would just require this to be defined first

```
type List struct {
    Items      []*unstructured.Unstructured `json:"items"`
    ApiVersion string `json:"apiVersion" yaml:"apiVersion"`
    Kind       string `json:"kind"`
    Metadata   map[string]any `json:"metadata"`
}

func NewList(objs []unstructured.Unstructured) *List {
    list := &List{
        ApiVersion: "v1",
        Kind: "List"
        Metadata: maps[string]any{},
    }
    for _, obj := range objs {
        list.Items = append(list.Items, &obj)
    }
    return list
}
```

```suggestion
		switch o.Output {
		case formatJson.String():
		    list = NewList(objs)
		    listData, err := json.MarshalIndent(list, "", "    ")
			if err != nil {
				return err
			}
			fmt.Println(string(listData))
			return nil
	    case formatYaml.String():
	        listYaml, err := yaml.Marshal(list)
			if err != nil {
				return err
			}
			fmt.Println(string(listYaml))
			return nil
		}
	}
```